src/common/hooks/account/use-account-transactions-with-transfers.hooks.ts

```diff
@@ -0,0 +1,43 @@
+import { useQuery, UseQueryOptions } from 'react-query';
+import { useCurrentAccountStxAddressState } from '@store/accounts/account.hooks';
+import { useCurrentNetworkState } from '@store/network/networks.hooks';
+import { useAccountsApi } from '@store/common/api-clients.hooks';
+import { AddressTransactionWithTransfers } from '@stacks/stacks-blockchain-api-types';
+import { useAccountTransactionsWithTransfersState } from '@store/accounts/transactions-with-transfer.hooks';
+import { DEFAULT_LIST_LIMIT, QueryRefreshRates } from '@common/constants';
+import { PaginatedResults } from '@common/types';
+
+const QUERY_OPTIONS = {
+  refetchInterval: QueryRefreshRates.MEDIUM,
+  refetchOnMount: 'always',
+  refetchOnReconnect: 'always',
+  refetchOnWindowFocus: 'always',
+};
+
+export function useGetAccountTransactionsWithTransfer(reactQueryOptions: UseQueryOptions = {}) {
```

Comment:
Can you coordinate the pluralization throughout your code overall so I know if this is 1 transfer or many? I still am unsure? I see it in different places used as singular and in other places as plural.